double-conversion/bignum.cc

```diff
@@ -26,17 +26,28 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include <algorithm>
+#include <cstring>
 
 #include "bignum.h"
 #include "utils.h"
 
 namespace double_conversion {
 
 Bignum::Bignum()
-    : bigits_buffer_(), bigits_(bigits_buffer_, kBigitCapacity), used_digits_(0), exponent_(0) {
-  for (int i = 0; i < kBigitCapacity; ++i) {
-    bigits_[i] = 0;
-  }
+    : used_digits_(0), exponent_(0) {
+  std::memset(bigits_buffer_, 0, sizeof(Chunk) * kBigitCapacity);
```

Comment:
TBH, I would expect a good compiler to optimize this away, but fine with it.

Reply:
This change was more a "one line of code is better than two" kind of thing, not intended as a performance optimisation.